frontend/src/sass/elements/_alerts.scss

```diff
@@ -12,3 +12,17 @@
 .usa-alert a.usa-button:hover {
   color: white;
 }
+
+.application-browser-alert {
+  margin-top: 0rem;
```

Comment:
Nothing to change here but we should maybe have a larger conversation with the team about our use of `px` `em` `%` `vh` `vw` and `rem` throughout the codebase. I like `em` and `rem` since they scale, but more than anything I think it would be good for us to be consistent with our usage.